src/basic/hostname-util.c

```diff
@@ -11,6 +12,21 @@
 #include "string-util.h"
 #include "strv.h"
 
+const char* get_fallback_hostname(void) {
+        const char *e;
+
+        e = secure_getenv("SYSTEMD_FALLBACK_HOSTNAME");
+        if (e) {
+                if (!hostname_is_valid(e, 0))
+                        log_debug("Invalid hostname in $SYSTEMD_FALLBACK_HOSTNAME, ignoring.");
+                else
+                        return e; /* The pointer is valid as long as a getenv(), putenv(), setenv(), or
```

Comment:
invert the conditions to keep indentation low and avoid unnecessary negation?

```c
e = …
if (e) {
        if (hostname_is_valid(e, 0))
                return e;
        log_debug(…);
}

return FALLBCAK_HOSTNAME;
}
```

Reply:
Yeah, that's nicer. I don't know why I didn't do that.